test/connect.test.ts

```diff
@@ -72,7 +72,7 @@ describe("connect function", function () {
         } as unknown as Signer // convince type checks into letting us mock the signer
       });
     } as ConnectionOptions).rejects.toThrow(
-      "Only Ethereum Rinkeby network is currently supported. Switch your wallet connection and reconnect."
+      "Only Ethereum Rinkeby, Optimism Kovan, and Local Hardhat networks are currently supported. Switch your wallet connection and reconnect."
```

Comment:
Can we have this be generated from the phrases in the contractAddresses const, so we aren't hard "coding" this sentence? Maybe we can just create a separate issue for that, rather than holding up a PR with something as simple as this?